src/components/Dialog.js

```diff
@@ -107,26 +108,43 @@ export default class Dialog extends Component {
     window.addEventListener('keydown', this.__onKeyDown, true);
   }
 
+  componentDidMount() {
+    this.closeBtnRef.focus();
+  }
+
   componentWillUnmount() {
     window.removeEventListener('keydown', this.__onKeyDown, true);
   }
 
   render() {
     return html`
       <div class="${styles}" onClick=${this.__onClick}>
-        <div ref=${(ref) => (this.dialogRef = ref)} class="ld-dialog--container">
-          <div class="ld-dialog--header">
-            <h1>${this.props.title}</h1>
-            <div class="ld-dialog--close-cont">
-              <button onClick=${this.__closeDialog} class="ld-dialog--close" aria-labelledby="button-name">
-                <span id="button-name" hidden>close</span>
-                ${close}
-              </button>
+        <div
+          role="dialog"
+          aria-labelledby="dialogtitle"
+          aria-modal="true"
```

Comment:
a modal is not closable on outside click (which is generally the best practice) should we change this also?

Reply:
So you mean remove the `aria-modal`, or remove the functionality that closes the dialog on outside click?

Reply:
Yeah I think we should do one of those two.

Reply:
I removed aria-modal